examples/deployment/scripts/transfer/master-fund-john.js

```diff
@@ -0,0 +1,17 @@
+const algosdk = require('algosdk');
+const {
+  printCreatedAsset,
+  printAssetHolding,
+  transferMicroAlgos,
+  asaOptIn,
+  transferAsset
+} = require('../../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
```

Comment:
same here, let's add some docs comments.

Reply:
added same as in previous one